pkg/config/config.go

```diff
@@ -38,6 +38,7 @@ func InitConfig(ctx context.Context) {
 	viper.SetDefault("logLevel", "info")
 	viper.SetDefault("cache.store", "bigcache")
 	viper.SetDefault("cache.expiry", "3600m")
+	viper.SetDefault("plugins.exporterDir", "/plugins")
```

Comment:
May be good to mention the config options in the README for the local.yaml

Reply:
Im rewriting this documentation in another branch, I will add it there. Its also still up in the air on how we want to document the plugins